nexusdialog/src/main/java/com/github/dkharrat/nexusdialog/validations/ValidationError.java

```diff
@@ -5,7 +5,7 @@
 /**
  * Represents a validation error of a form's input field.
  */
-public abstract class ValidationError {
+public abstract class ValidationError extends Exception {
```

Comment:
I was using the `Exception` mechanism to add the meaning "unexpected input from the user", but you're right, this should'nt be used here.